.goreleaser.ubuntu-latest.yml

```diff
@@ -34,3 +34,5 @@ archives:
     files:
       - LICENSE.md
       - README.md
+checksum:
+  name_template: "{{ .ProjectName }}_{{ .Version }}_linux_windows_checksums.txt"
```

Comment:
Would it make sense to just use the `"{{ .ProjectName }}_{{ .Version }}_checksums.txt"` to cater for Windows and Linux?

CC: @wolfeidau

Reply:
ok, I've made the change